service/git_context.ml

```diff
@@ -65,14 +66,16 @@ let filter_available t pkg opam =
         (OpamFilter.to_string available);
       Error Unavailable
 
-let version_compare (v1, opam1) (v2, opam2) =
+let version_compare t (v1, opam1) (v2, opam2) =
```

Comment:
```suggestion
let version_compare lower_bound (v1, opam1) (v2, opam2) =
```

because the function doesn't need all of `t`, just the `lower_bound` parameter